modules/setting/markup.go

```diff
@@ -44,7 +44,7 @@ func newMarkup() {
 			continue
 		}
 
-		if name == "sanitizer" {
+		if name == "sanitizer" || strings.HasPrefix(name, "sanitizer.") {
```

Comment:
Is it a requirement that the sections have different names? Can we detect that here or they will be mixed together by the ini library if the user repeats names?

Reply:
They have to have different names. Go-ini does not provide a mechanism for the detection of repeat sections.